modules/runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/command/PositionCommand.java

```diff
@@ -280,4 +292,38 @@ protected LogOffset searchWatermarkOffset(LogTailer<Record> tailer, long timesta
         return lastOffset;
     }
 
+    protected boolean positionToOffset(LogManager manager, Name group, Name name, int partition, long offset) {
+        LogLag lag = getLag(manager, group, name, partition);
+        long pos = lag.lower();
+        boolean goToStart = false;
+        if (offset < pos) {
+            goToStart = true;
+        } else if (offset == pos) {
+            log.error(String.format("Current offset for group %s on %s is already %d", group, labelFor(name, partition),
+                    pos));
+            return false;
+        }
+        try (LogTailer<Externalizable> tailer = manager.createTailer(group, LogPartition.of(name, partition))) {
+            if (goToStart) {
+                tailer.toStart();
+            }
+            for (;;) {
+                LogRecord<Externalizable> record = tailer.read(Duration.ofSeconds(1));
+                if (record == null || record.offset().offset() > offset) {
+                    log.error("No offset found for the specified partition");
+                    return false;
+                }
+                if (record.offset().offset() == offset) {
+                    tailer.commit();
+                    log.warn(String.format("# Move group %s on %s from %d to %d", group, labelFor(name, partition), pos,
+                            offset));
+                    return true;
+                }
+            }
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            log.error("Interrupted");
```

Comment:
Do we want:
```
log.error("Interrupted", e);
```
?

Reply:
No I don't think so, interrupts aren't error so the location that caught the exception doesn't matter. I'm not sure why we would log at all to be honest.

Reply:
This is the output of a command-line tool, not trace of a Nuxeo Service, the output can be copy/pasted for support purpose and if someone hit Ctrl-C It is important to know that the command was interrupted, the stack trace is useless.